packages/core/types/index.js

```diff
@@ -271,33 +271,55 @@ export type SourceLocation = {|
 export type Meta = JSONObject;
 
 export type Symbol = string;
-export interface Symbols // eslint-disable-next-line no-undef
+export interface AssetSymbols // eslint-disable-next-line no-undef
   extends Iterable<[Symbol, {|local: Symbol, loc: ?SourceLocation|}]> {
   get(exportSymbol: Symbol): ?{|local: Symbol, loc: ?SourceLocation|};
   hasExportSymbol(exportSymbol: Symbol): boolean;
   hasLocalSymbol(local: Symbol): boolean;
-  // Whether static analysis bailed out
-  +isCleared: boolean;
+  exportSymbols(): Iterable<Symbol>;
 }
-export interface MutableSymbols extends Symbols {
-  // Static analysis bailed out
```

Comment:
In hindsight, having cleared symbols on an asset doesn't make any sense.
A symbol `*:$id$exports` can be used as a fallback instead (this exists on CommonJS and not on ES6 modules)